tests/matrix_free/mass_operator_01.cc

```diff
@@ -116,7 +116,7 @@ void test ()
     mf_data.reinit (dof, constraints, quad, data);
   }
 
-  MatrixFreeOperators::MassOperator<dim,fe_degree, 1, number> mf;
+  MatrixFreeOperators::MassOperator<dim,fe_degree, fe_degree+1, 1, number> mf;
```

Comment:
I would prefer to use `fe_degree+2` here and below as well. I remember to have observed negative values for the lumped mass matrix for `FE_Q_Hierarchical`.